### ISSUE TITLE
Use result in size property

### ISSUE THREAD

**Issue by simonschllng** [jfjlaros/bin-parser#3]
block_name_length returns 7. Using it in size below throws an error. How can this be achieved correctly?

````
- name: CEmbOne
  structure:
  - name: block_name_length
    type: int
    size: 2
  - name: block_name
    type: text
    size: 7 #block_name_length
````

**Comment by jfjlaros** [comment]
I introduced [this functionality]( https://github.com/jfjlaros/bin-parser/commit/f77906dd917ccf4629176c0c25a6c32377f4c373) just a few days ago. I did not have the chance to document it or release it via PyPI yet though.

**Comment by jfjlaros** [comment]
This functionality is now available in release version 0.0.13.
